qa/tasks/ceph.conf.template

```diff
@@ -14,7 +14,6 @@
 	ms die on bug = true
 
 	mon pg warn min per osd = 1
-	mon pg warn max per osd = 10000   # <= luminous
```

Comment:
@neha-ojha after a second thought, i am wondering if we should keep this option if we are doing downgrade tests..

Reply:
Might make sense to get rid of it in master at least, since this option got removed in luminous and we care about mimic and beyond upgrades to master.

Reply:
Thanks, Neha!